tests/functionnal/ITILFollowup.php

```diff
@@ -207,4 +207,53 @@ public function testUpdateAndDelete() {
       )->isTrue();
       $this->boolean((boolean) $fup->getFromDB($fup_id))->isFalse();
    }
+
+   /**
+    * Test _do_not_compute_takeintoaccount flag
+    */
+   public function testDoNotComputeTakeintoaccount() {
+      $this->login();
+
+      $ticket = new \Ticket();
+      $oldConf = [
+        'glpiset_default_tech'      => $_SESSION['glpiset_default_tech'],
+        'glpiset_default_requester' => $_SESSION['glpiset_default_requester'],
+      ];
+
+      $_SESSION['glpiset_default_tech'] = 0;
+      $_SESSION['glpiset_default_requester'] = 0;
+
+      // Normal behaviior, no flag specified
+      $ticketID = $this->getNewITILObject('Ticket');
+
+      $ITILFollowUp = new \ITILFollowup();
+      $ITILFollowUp->add([
```

Comment:
You should test result of all add, update, getFromDB etc; to make sure each step is working as expected